doc/nixcon.md

```diff
@@ -43,3 +43,74 @@ Organisers are using:
+
+## Sponsorship tiers
+
+This should be displayed on the website.
+
+### Non-sponsorship donation (0+ EUR)
+- Supporting the conference and official Nix projects
+- Confirmation of donation via OpenCollective
+
+### Bronze (1000+ EUR):
+- Linked logo on the website
+- Linked shout-out or repost from official social media accounts
+- Possibility to distribute your own stickers/swag/merch at the venue
+- Tickets: 1 corporate tickets[^1]
+
+[^1]: Only corporate tickets come with company name/logo recognition on the badge. Corporate tickets can also be bought individually when ticket sales open.
+
+### Silver (3000+ EUR)
+- Everything from Bronze
+- Shout-out in the opening
+- Slot on break slideshow (provide a slide in advance)
+- Possibility to distribute flyers (provide them in advance) to all attendees in bag
+- Tickets: 3 corporate tickets[^1]
+
+### Gold (5000+ EUR):
+- Everything from Silver
+- 5 minute lightning talk slot
+- Dedicated space for a booth[^2]
+- Tickets: 5 corporate tickets[^1]
+
+[^2]: These perks may be limited in availability.
+
+### Diamond (10000+ EUR):
+- Everything from Gold
+- Logo on some official conference swag[^2][^3]
+- 1 minute presentation slot (provide the slides in advance) in the conference opening
+- Intro of all recordings show your name/logo
+- Possibility to distribute items (provide them in advance) to all attendees in bag
+- Tickets: 10 corporate tickets[^1]
+
+[^3]: These perks require production and cannot be guaranteed if too late.
+
+### General info
+- Sponsors can be tier-limited or rejected by the Nix Steering Committee for any reason.
+- The outline of all sponsor content must be disclosed in advance. We reserve the right to reject content.
+- Visibility priority is given according to tiers and donation amount.
+- The sponsor can selectively abstain from perks.
+- Surplus funds will be used to support official Nix projects.
+- Reach out if you'd like to sponsor in a more custom way, such as food, drinks or only specific perks.
+
+### How to sponsor
+
+Send an email to sponsor@nixos.org (which will forward to both the Nix Steering Committee and the NixOS Foundation board) with:
+- Company name and website
+- Desired tier, amount and perks
+- Any other details
+
+## Sponsorship process
+
+The following is internal documentation and should not go on the website. It involves the following respondents:
+- Board respondent: @infinisil or @ra33it0?
+- SC respondent: ???
+
+1. After receiving the email, board respondent replies that the email was received and that they will hear back within about 2 weeks.
+2. SC respondent ensures the SC internally rejects or provisionally approves the sponsor within 1 week.
+   - If rejected, SC respondent replies to the sponsor with the rejection message.
+3. If provisionally approved by the SC, SC respondent forwards the name and website of the sponsor to the [NixCon 2025 orga Matrix room](https://matrix.to/#/%23nixcon2025-orga:lassul.us) for organiser approval.
```

Comment:
Why is this written year-dependent? Would be better to make it future-proof while we're at it

Reply:
> Why is this written year-dependent? Would be better to make it future-proof while we're at it

Is the same matrix room being re-named and re-used each year?